verilog/analysis/verilog_linter.cc

```diff
@@ -141,6 +144,12 @@ void VerilogLinter::Configure(const LinterConfiguration& configuration) {
   for (auto& rule : syntax_rules) {
     syntax_tree_linter_.AddRule(std::move(rule));
   }
+
+  // Apply external waivers
+  if (FLAGS_waivers_file.IsModified()) {
+    lint_waiver_.ApplyExternalWaivers(configuration.ActiveRuleIds(),
+                                      absl::GetFlag(FLAGS_waivers_file));
+  }
```

Comment:
Recommend against accessing global flag variables here, but instead passing in results of those flags.

Reply:
Moved this to where the remaining flags are handled.